### PR TITLE
Add KerasNet validation summary UT

### PR DESCRIPTION
**Pull request by Litchilitchy**

### CODE REVIEW

File: pyzoo/test/zoo/models/recommendation/test_neuralcf.py [hunk 1]
```diff
@@ -80,6 +80,8 @@ def gen_rand_user_item_feature(user_num, item_num, class_num):
         model.fit(data, nb_epoch=1, batch_size=32, validation_data=data)
         train_loss = model.get_train_summary("Loss")
         val_loss = model.get_validation_summary("Loss")
+        val_acc = model.get_validation_summary("Accuracy")
+        print(np.array(val_acc))
```
Comment (qiuxin2012) [inline pyzoo/test/zoo/models/recommendation/test_neuralcf.py:84]: assert is not empty.

**Comment by qiuxin2012** [comment]
AUC  ->    AucScore
Accuracy   BinaryAccuracy CategoricalAccuracy  ->
HitRatio -> HitRatio@{k}  k is your top k.
Loss -> Loss
MAE -> MAE
MeanAveragePrecision in BigDL -> MAP@k k is your top k.
MeanAveragePrecision in Zoo -> PascalMeanAveragePrecision.
MeanAveragePrecisionObjectDetection -> MAPObjectDetection.
NDCG -> NDCG.
